<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeometryDialog</class>
 <widget class="QDialog" name="GeometryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NeXtRAD Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>460</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Geometry Settings</string>
   </property>
  </widget>
  <widget class="QLabel" name="n0_lat_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Node 0 Latitude (deg):</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="n0_lat_Edit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="n0_lon_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Node 0 Longitude (deg):</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="n0_lon_Edit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="n0_ht_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Node 0 Height (m):</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="n0_ht_Edit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>150</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="n1_lon_Edit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>240</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="n1_ht_Edit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>280</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="n1_ht_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Node 1 Height (m):</string>
   </property>
  </widget>
  <widget class="QLabel" name="n1_lat_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Node 1 Latitude (deg):</string>
   </property>
  </widget>
  <widget class="QLabel" name="n1_lon_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Node 1 Longitude (deg):</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="n1_lat_Edit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>200</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="n2_lon_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>370</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Node 2 Longitude (deg):</string>
   </property>
  </widget>
  <widget class="QLabel" name="n2_ht_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>410</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Node 2 Height (m):</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="n2_lat_Edit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>330</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="n2_lon_Edit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>370</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="n2_lat_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>330</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Node 2 Latitude (deg):</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="n2_ht_Edit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>410</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GeometryDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GeometryDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
